src/app/templates/speech_recognition_local.sh

```diff
@@ -0,0 +1,14 @@
+{% extends "base_local.sh" %}
+{% block command %}
+{%- if container_config.provider == "docker" %}
```

Comment:
We also want a branch for case `container_config.provider == "apptainer"` because that is what we would expect on (e.g.) Della.

Reply:
Does the local slum script means that both the Blackfish API and service API would be run on the same place, e.g. Della or Local Laptop? Would change it as previously I thought that local means that the APIs would run merely on local laptop.

Reply:
Yes—"local" just means both API are running on the same machine.